javadoc/pom.xml

```diff
@@ -234,6 +234,16 @@
                            <artifactId>c3p0</artifactId>
                            <version>0.9.5-pre8</version>
                         </additionalDependency>
+                        <additionalDependency>
+                           <groupId>com.zaxxer</groupId>
+                           <artifactId>HikariCP</artifactId>
+                           <version>${version.hikaricp}</version>
+                        </additionalDependency>
+                        <additionalDependency>
+                           <groupId>org.jboss.slf4j</groupId>
```

Comment:
SLF4J API should probably be here; `slf4j-jboss-logging` is only a bridge between implementations.